src/main/java/com/jvm_bloggers/core/newsletter_issues/domain/NewsletterIssue.java

```diff
@@ -37,7 +36,7 @@
 @Builder
 @Getter
 @ToString(exclude = {"blogPosts", "newBlogs"})
-public class NewsletterIssue {
+public class NewsletterIssue implements NewsletterIssueBaseData {
```

Comment:
@tdziurko In the other hand, it's just the interface used for DB query results projection. Current implementation benefit is that changes to model also would impact interface.

Maybe we could just rename this interface to "NewsletterIssueSimple"?